cs464-group-project/src/components/navigationBar/NavBar.js

```diff
@@ -6,13 +6,13 @@ import '../../style/NavigationBar.css';
 function NavBar() {
   return (
     //https://react-bootstrap.netlify.app/docs/components/navbar
-    <Navbar expand='lg' className='navbar-container'>
-      <Navbar.Brand href='/'>Football Stats</Navbar.Brand>
-      <Navbar.Toggle aria-controls='basic-navbar-nav' />
-      <Navbar.Collapse id='basic-navbar-nav'>
-        <Nav className='me-auto'>
-          <Nav.Link href='/teamsmainpage'>Team</Nav.Link>
-          <Nav.Link href='/player'>Player</Nav.Link>
+    <Navbar expand="lg" className="navbar-container">
+      <Navbar.Brand href="/">Football Stats</Navbar.Brand>
+      <Navbar.Toggle aria-controls="basic-navbar-nav" />
+      <Navbar.Collapse id="basic-navbar-nav">
+        <Nav className="me-auto">
+          <Nav.Link href="/teamsmainpage">Team</Nav.Link>
+          <Nav.Link href="/player">Player</Nav.Link>
```

Comment:
in your Prettier setting, go ahead and check single quotes options.

Reply:
Switched to single quotes